apps/finance/app/src/components/Transfers.js

```diff
@@ -58,6 +89,47 @@ class Transfers extends React.PureComponent {
       ...initialState,
     })
   }
+  handleDateRangeChange = selectedDateRange => {
+    this.setState({ selectedDateRange })
+  }
+  encodeDataToCsv = (data, tokenDetails) => {
+    const csvContent = [
+      'data:text/csv;charset=utf-8,Date,Source/Recipient,Reference,Amount',
+    ]
+      .concat(
+        data.map(
+          ({
+            date,
+            numData: { amount },
+            reference,
+            isIncoming,
+            entity,
+            token,
+          }) => {
+            const { symbol, decimals } = tokenDetails[toChecksumAddress(token)]
+            const formattedAmount = formatTokenAmount(
```

Comment:
Wondering if we should use the `BN.js` numbers here instead; `formatTokenAmount()` is **NOT** meant to be used for accounting software due to precision issues.

Reply:
Created an issue https://github.com/aragon/aragon-apps/issues/756

Please add missing info :)